src/styles/page.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $block-gap;
+  padding: $calendar-padding;
+  width: width-calendar($day-size);
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    box-sizing: border-box;
+    background: $block-bg;
+    width: $day-size;
+    height: $day-size;
+    border: 1px solid #000;
+    transition-duration: 0.5s;
```

Comment:
add transition property for those rules you change

Reply:
<img width="648" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/70220667/fc04b8b9-62cd-4f82-acde-457dbd7a11a1">